source/_patterns/01-molecules/content-header/content-header.twig

```diff
@@ -4,6 +4,40 @@
 
   {%- block body -%}
 
+    {% if subjects.items is iterable %}
```

Comment:
We'd want to check if the list is empty or not. But this should be done inside a `{% block %}` anyhow (to allow the whole thing to be overridden).

Reply:
A `{%- block categories -%}` has been added and the conditional `{% if items is iterable and items|length > 0 %}` the condition is only met when `categories.items` is iterable and not empty.